src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * The Ui class deals with the interactions with the user.
+ */
+public class Ui {
+    private Scanner sc;
+
+    /**
+     * Constructor for a Ui object.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Greets the user.
+     */
+    public String greet() {
+        return ("Hello, I'm Bo-Chan!\n" + "How may I help you?");
+    }
+
+    /**
+     * Says bye to the user.
+     */
+    public String exit() {
+        return "Bye, see you soon!";
+    }
+
+    /**
+     * Scans the input of the user.
+     * @return a String object of the user input.
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints the message when a task is successfully added into the tasklist.
+     * @param task Task added into the tasklist.
+     * @param taskList tasklist of the user.
+     */
+    public String showAddition(Task task, TaskList taskList) {
+        return ("Added this task to your list:\n" + task + "\n" + showTaskTotal(taskList));
+    }
+
+    /**
+     * Prints the message when a task is successfully completed.
+     * @param task Task completed.
```

Comment:
Perhaps you should leave a line between your description and your @param line?

Reply:
Alright thanks